sub-packages/bionemo-geometric/src/bionemo/geometric/molecule_featurizers.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-Apache2
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import torch
+from rdkit.Chem import Descriptors, Mol
+
+from bionemo.geometric.base_featurizer import (
+    BaseMoleculeFeaturizer,
+)
+
+
+class RDkit2DDescriptorFeaturizer(BaseMoleculeFeaturizer):
```

Comment:
Would recommend adding docstring on the `get_molecule_features()` with more details about what the requirements are for the input and what to expect in the output. Would be great if an example is give in the class's docstring